membrane_curvature/base.py

```diff
@@ -17,8 +17,9 @@
 
 import MDAnalysis
 from MDAnalysis.analysis.base import AnalysisBase
+from MDAnalysis.transformations import wrap
```

Comment:
This isn't being used as far as I can see? The wrap method on AtomGroups requires no import.